### PR TITLE
Chore/ansible cleanup

### PR DESCRIPTION
**Pull request by goeland86**
Cleaned out some of the unused roles, moved the replicape-only stuff into a legacy folder, and moved unused but potentially useful roles into an addons folder.

The main playbooks are simplified by using the meta/main.yml to define installation dependencies between roles (i.e. weston can't install until linux_preparation has run, mainsail needs moonraker, etc.).

### CODE REVIEW

File: roles/install_klipperscreen/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+dependencies:
+  - { role: install_weston }
```
Comment (eliasbakken) [inline roles/install_klipperscreen/meta/main.yml:3]: Weston is not needed for KlipperScreen.

File: roles/install_klipper/tasks/recore.yml [hunk 2]
```diff
@@ -218,18 +208,3 @@
...
-- name: Recore - make sure the sudoers.d/debian file exists
-  file:
-    path: /etc/sudoers.d/debian
-    owner: root
-    group: root
-    state: touch
-
-- name: Recore - create folder for klipper logs in /var/log
-  file:
-    path: /var/log/klipper_logs
```
Comment (eliasbakken) [inline roles/install_klipper/tasks/recore.yml:231]: This looks like a problem on OctoPrint. Klipper will not start: 
FileNotFoundError: [Errno 2] No such file or directory: '/var/log/klipper_logs/klippy.log'

**Comment by goeland86** [comment]
Ah. I think that's probably a change in the picturing config file we can do
to point it at the Right location (
/home/debian/printer_data/logs/klippy.log).

I'll push them change to the pr and build an image locally to test.

File: roles/install_klipper/tasks/recore.yml (hunk 2 above)
Comment (goeland86) [reply roles/install_klipper/tasks/recore.yml:231]: Will revert this. I hadn't thought of the need to store the log in zram. I'll also add symlinks in the linux_preparation stage for the entire `/home/debian/printer_data/logs/` folder to be in `/var/log/` as well.

**Comment by eliasbakken** [comment]
I think the klipper systemd file already puta the log in /var/log/klippy
dosent it?

tir. 27. jun. 2023, 20:30 skrev Jon C ***@***.***>:

**Comment by goeland86** [comment]
> I think the klipper systemd file already puta the log in /var/log/klippy dosent it?

Yes. However moonraker, mainsail, etc. all store their logs to `/home/debian/printer_data/logs`. What I did is mainly to ensure that the symlink exists, so that we don't accidentally get some logs in the home directory instead of zram.

**Pull request merged**